### PR TITLE
Add section for repeat action

### PR DESCRIPTION
**Pull request by stirlhoss**
This is just an addition that I made to the tutor file. I want to do more of these but I want to get one submitted so I know if there are things I need to change or do better with. 

I added repeating actions to section 4 and updated the section 4 summary.

### CODE REVIEW

**Comment by the-mikedavis** [comment]
It looks like @n0s4 is covering these in the big update in #2798: https://github.com/helix-editor/helix/pull/2798/files#diff-0751fcdc9d0b2b023e5c02ee4941114ca1706d04e1ba73c2f4c5ba287c847074R661-R674

`.` is actually to repeat the last insert rather than the last action which was a common point of confusion that has recently been fixed in the docs: https://github.com/helix-editor/helix/pull/2878

**Comment by stirlhoss** [comment]
Made updates with taking your comments into consideration. It is difficult to come up with examples for this feature that aren't either too complicated or too basic. I went basic this time. I think that functions like this are things that once you get a feel for when you can use it, it sort of just clicks. Let me know what you guys think.

File: runtime/tutor.txt [hunk 1]
```diff
@@ -504,6 +504,36 @@ _________________________________________________________________
...
+3. Go to the next instance and select the word again.
+4. Type .
+5. Move to the next and repeat until they all say 'Orange'.
+
+--> Banana Banana Banana Banana Banana Banana 
+
+To repeat motion:
+
+1. Move the Cursor to the first line below marked -->.
+2. Press 'f' then 'O' to find the next instatne of 'O'
```
Comment (pythoneer) [inline runtime/tutor.txt:530]:
```suggestion
2. Press 'f' then 'O' to find the next instance of 'O'
```

**Comment by archseer** [comment]
Ended up merging https://github.com/helix-editor/helix/pull/2798

**Pull request closed without merging**